src/NServiceBus.Core/Recoverability/FirstLevelRetries/FirstLevelRetriesBehavior.cs

```diff
@@ -42,7 +40,6 @@ class FirstLevelRetriesBehavior : Behavior<ITransportReceiveContext>
                 {
                     storage.ClearFailuresForMessage(pipelineUniqueMessageId);
                     context.Message.Headers[Headers.FLRetries] = numberOfFailures.ToString();
-                    notifications.Errors.InvokeMessageHasFailedAFirstLevelRetryAttempt(numberOfFailures, context.Message, ex);
```

Comment:
discusses this with @andreasohlund and we decided to fix the naming of the FLR notification events on this (already existing) PR: https://github.com/Particular/NServiceBus/pull/3568